Cargo.toml

```diff
@@ -72,7 +72,7 @@ ethers-middleware = { git = "https://github.com/gakonst/ethers-rs", default-feat
 ethers-etherscan = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
 ethers-solc = { git = "https://github.com/gakonst/ethers-rs", default-features = false }
 
-solang-parser = "=0.3.0"
+solang-parser = "=0.3.1"
```

Comment:
I bump solang-parser due to the change of it in [ethers-solc](https://github.com/gakonst/ethers-rs/blob/master/ethers-solc/Cargo.toml#L31). 

However, this introduces some compilation errors resulting in some minor modifications in chisel and fmt.

Reply:
is updated on master:

https://github.com/foundry-rs/foundry/blob/044f680a5c2541b0fdbd69eb8f33abd57e161569/Cargo.toml#L75-L75

Reply:
@mattsse  rebased.